src/common/LogClient.cc

```diff
@@ -332,6 +335,15 @@ version_t LogClient::queue(LogEntry &entry)
   return entry.seq;
 }
 
+void LogClient::reset()
+{
+  std::lock_guard l(log_lock);
+  if (log_queue.size()) {
```

Comment:
nit, might want to use `!log_queue.empty()`, but I think it's fine if we just call `log_queue.clear()` without checking if `log_queue` is empty. as this is not a critical path, and the overhead of calling `clear()` is relatively low.